app/controllers/web/admin/news_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Web::Admin::NewsController < Web::Admin::ApplicationController
+  def index
+    @published_news = NewsDecorator.decorate_collection News.published.order('published_at DESC') #FIXME using another decorator in that file
+    @unpublished_news = NewsDecorator.decorate_collection News.unpublished.order('published_at DESC')
```

Comment:
Line is too long. [101/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.